bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/AUF2.java

```diff
@@ -173,7 +178,27 @@ public void dragSetData(DragSourceEvent event) {
 				event.data = sb.toString().replace(",$", StringUtils.EMPTY); //$NON-NLS-1$
 			}
 		});
-
+		tv.addSelectionChangedListener(new ISelectionChangedListener() {
+			@Override
+			public void selectionChanged(SelectionChangedEvent event) {
+				selectedCertificates.clear();
+				IStructuredSelection selection = (IStructuredSelection) tv.getSelection();
+				for (Object obj : selection.toArray()) {
+					if (obj instanceof ISickCertificate) {
+						selectedCertificates.add((ISickCertificate) obj);
+					}
+				}
+			}
+		});
+		tv.getTable().addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseDown(MouseEvent e) {
+				if (tv.getTable().getItem(new Point(e.x, e.y)) == null) {
+					tv.setSelection(StructuredSelection.EMPTY);
+					selectedCertificates.clear();
+				}
+			}
+		});
```

Comment:
Weshalb ist dierser Code nötig? Soll immer nur 1 Element im Viewer selektiert sein?

Reply:
Ja, der Code wird benötigt. Wenn du etwas auswählst und dann in einen leeren Bereich klickst, wird die Auswahl wieder aufgehoben. Das ist beim Drucken sehr nützlich, besonders wenn du die ganze Liste drucken möchtest. Indem du einfach ins Leere klickst, kannst du sicherstellen, dass nichts ausgewählt ist. Ohne diesen speziellen Code wäre es nicht möglich, eine Auswahl wieder abzuwählen.